packit/cli/init.py

```diff
@@ -105,3 +106,21 @@ def generate_config(
         logger.debug(f"Packit config file '{config_file}' changed.")
 
     return output_config
+
+
+@click.command("generate", context_settings=get_context_settings())
+@click.argument("path_or_url", type=LocalProjectParameter(), default=os.path.curdir)
+@click.option(
+    "-f", "--force", is_flag=True, help="Reset config to default if already exists."
+)
+@cover_packit_exception
+@click.pass_context
+def generate(ctx, path_or_url, force):
```

Comment:
This is not the optimal solution since it violates the DRY principle. If we even need to fix an issue in the code, we need to do it on two place, instead of one.

Click docs provide a better way to implement aliases for commands: http://click.palletsprojects.com/en/5.x/advanced/#command-aliases

Reply:
Aliasing is possible, but we actually don't want only an alias -- we want a slightly different code for both. (I am not sure if we can get that info from the context (`ctx`)?

- We want a deprecation warning.
- If we add some other non-generation function to `init`, do we want to have it also in the `generate`? (=Is this only a deprecated alias or a deprecated subcommand that will not get any new options.)

Reply:
I hold my opinion: it should be an alias. But good point with the deprecation warning.

Anyway, no matter how it gets implemented, both commands should share as much code as possible.

Reply:
Yes, aliasing is possible I will try out a few things. I got an idea to include a deprecation warning, I will comment if I get stuck.